incipyt/system.py

```diff
@@ -58,6 +58,26 @@ def pull(self, key):
 
         return self._variables[key]
 
+    def pull_keys(self, keys, sanitizer=None):
+        """Pull multiple `keys` at once and sanitize them.
+
+        See also :func:`incipyt.system.Environment.pull`, which will be used to
+        pull each key from the environment.
+
+        :param keys: Requiered environment keys. If a key is `None`, it will
+        not be pulled.
+        :type keys: :class:`collections.abc.Sequence`
+        :param sanitizer: Will be called on key-value pairs to sanitize values.
+        :type sanitizer: :class:`function`
+        :return: Sanitized environment key-value pairs.
+        :rtype: :class:`dict`
+        """
+        return {
+            key: sanitizer(key, self.pull(key)) if sanitizer else self.pull(key)
+            for key in keys
+            if key is not None
+        }
```

Comment:
@fblanchetNaN I'd appreciate your feedback on https://github.com/NotANameServer/incipyt/pull/15/commits/0e349227b0d896bdcd297420adf0925b658cf6ea :)

Reply:
Seems good to me.